libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/GovCert.hs

```diff
@@ -22,8 +23,13 @@ import Lens.Micro
 import Test.Cardano.Ledger.Constrained.Conway.Instances
 import Test.Cardano.Ledger.Constrained.Conway.PParams
 
-vStateSpec :: Specification fn (VState (ConwayEra StandardCrypto))
-vStateSpec = TrueSpec
+vStateSpec :: IsConwayUniv fn => Specification fn (VState (ConwayEra StandardCrypto))
+-- vStateSpec = TrueSpec
+vStateSpec = constrained' $ \x y z ->
+  [ assert $ sizeOf_ x ==. 1
+  , match y $ \cs -> assert $ sizeOf_ cs ==. 1
+  , assert $ z ==. lit (EpochNo 4)
```

Comment:
Why is the number of dormant epochs set to 4?

Reply:
The other constraints also look like they are overconstraining the value too much.

Reply:
So I have changed these to

  [ assert $ sizeOf_ dreps >=. 1
  , match commstate $ \cs -> assert $ sizeOf_ cs <=. 5
  , assert $ dormantepochs >=. lit (EpochNo 4)
  ]

If they get too big, it is impossible to debug things when something goes wrong I hope this is a nice compromise.